pkg/services/alerting/notifiers/telegram.go

```diff
@@ -48,6 +48,14 @@ func init() {
 				PropertyName: "chatid",
 				Required:     true,
 			},
+			{
+				Label:        "Message Thread ID",
+				Element:      alerting.ElementTypeInput,
+				InputType:    alerting.InputTypeText,
+				Description:  "Integer Telegram Message Thread Identifier",
+				PropertyName: "messagethreadid",
+				Required:     false,
+			},
```

Comment:
this is not needed as it is related to the legacy alerting that will be deleted soon

Reply:
I reverted the file 👍